_lessons/polish-project.html

```diff
@@ -50,51 +68,126 @@
 [github.com/fishtown-analytics/dbt/releases](https://github.com/fishtown-analytics/dbt/releases)
 
 ???
-
-* `require-dbt-version`: will help ensure that everyone is using the same version of dbt, especially important for CLI users
-
+""  
+It's best practice to require a dbt version to prevent conflicts. We do this by  
+setting a configuration called `require-dbt-version`.  
+  
+In our example, our project works with versions later than or equal to 0.17.1  
+and less than 0.19.0. By requiring this, you're ensuring other developers are  
+running the project with a version that can handle the functionality used within  
+it.  
+  
+This is more important for CLI users because dbt Cloud uses a development  
+environment, which configures the version that developers use when they access  
+the `develop` tab to make changes.  
+""  
+  
+[Teacher Notes:]  
+You can take a moment here to exemplify what happens when you add this config.  
+You should hit these points:  
+- How does a dbt run function when your version is within the requirements?
+- How does a dbt run function when your version is out of the requirements?
+  
+You'll need to click on the status in the bottom right corner and scroll all  
+the way down the error message to see the message where the version doesn't  
```

Comment:
suggestion: we don't _always_ make this super clear, would this be worth escalating into a demo??